src/lib/components/ui/input.svelte

```diff
@@ -74,6 +82,16 @@
 					</label>
 				{/if}
 			</div>
+			{#if search}
+				<div class="delete-button">
```

Comment:
I don't think this needs a specific class
```suggestion
				<div class="control-buttons">
```